ext/pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[build-system]
+build-backend = "hatchling.build"
+requires = ["hatchling"]
+
+[project]
+name = "django-stubs-ext"
+# NB! For clarity, keep version major.minor.patch in sync with django-stubs.
+# It's fine to skip django-stubs-ext releases, but when doing a release, update this to newest django-stubs version.
+version = "5.2.0"
+description = "Monkey-patching and extensions for django-stubs"
+readme = "README.md"
+license = "MIT"
+license-files = ["LICENSE.md"]
+maintainers = [
+  { name = "Marti Raudsepp", email = "marti@juffo.org" },
+]
+authors = [
+  { name = "Simula Proxy" },
+]
+requires-python = ">=3.10"
+classifiers = [
+    "License :: OSI Approved :: MIT License",
```

Comment:
```suggestion
  "License :: OSI Approved :: MIT License",
```

Reply:
and all others should have a proper spacing

Reply:
Should I use https://github.com/tox-dev/pyproject-fmt ? I can add it to the pre-commit config so that this gets handled automatically

Reply:
IFF it will not format headers, because it might turn 

```toml
[tool.ruff.lint.per-file-ignores]
```

into

```toml
[tool.ruff]
lint.per-file-ignores."tests/*.py" = ["INP001", "PGH003"]
```

which is very hard to read.

If it is configurable / disabled - sure!

Reply:
It doesn't seem configurable so I've just updated to spacing to 2 spaces everywhere